### DISCUSSION TITLE
Which role is allowed to manage frameworks ?

### DISCUSSION THREAD

**Discussion by nidhikachhawa24** [Sunbird-Ed/Community#250] [Documentation]
Suppose, we want to make changes in the framework, then can any role make changes in the framework?
Or is there any specific role who can manage the framework changes?

cc: @rajeevsathish @reshmi-nair @Krishnaj20

**Comment by Krishnaj20** [comment]
@nidhikachhawa24 @coolbung - There are multiple new discussion threads initiated in the last 24 hrs around framework management and channels. Can you articulate what are you trying to do here? With that context we might be able to help you address your queries. Also, have you gone thru the [ED](https://ed.sunbird.org/learn/readme) microsite? The relevant knowlg microsite links are also available in the ED site. I'd recommend you go thru the microsite. 
@sharathkashyap @vinukumar-vs

**Reply by nidhikachhawa24** [reply]
Hello @Krishnaj20 
It is in relation to the Framework management UI. We wish to understand the access control implemented for Framework APIs.

**Reply by Krishnaj20** [reply]
@nidhikachhawa24 - I'm copy pasting @sharathkashyap 's response in the other thread. 
"_Kindly look into the https://knowlg.sunbird.org/learn/product-and-developer-guide/taxonomy-and-tagging which explains the significance of Object Category, Taxonomy in the system. Since these are tenant and root level settings which are done at t requires the user to be have "ADMIN" role."

@vinukumar-vs

**Reply by Krishnaj20** [reply]
@nidhikachhawa24 - @vinukumar-vs will be the sponsor from Knowlg building block to help you complete the tech design of this module. Please feel free to reach out to him. He will help in completing the tech design which can then be shared for DC & community review/approval.

**Reply by chitranshu-keshav** [reply]
@nidhikachhawa24 @vinukumar-vs Can you please add the link of the JIRA issue related to this thread.

cc: @Krishnaj20
